build.gradle

```diff
@@ -37,6 +37,6 @@ repositories {
 }
 
 wrapper {
-    gradleVersion = '7.3.1'
+    gradleVersion = '8.3.0'
```

Comment:
Jeg tror riktig versjon bare er 8.3
Har du også kjørt ./gradlew wrapper for å oppdatere gradle wrapper jaren i prosjektet? 🙂  Hvis ikke så vil ikke denne endringen som er gjort her ha noen effekt. Du må kjøre ./gradlew wrapper to ganger, av en eller annen grunn

Reply:
ahh okey kan endre, men ja jeg kjørte ./gradlew wrapper kun en gang 😅